### PR TITLE
[pvr addons] Sync with latest kodi-pvr

### PR DESCRIPTION
**Pull request by hudokkow**
Sync remaining pvr addons with API 1.9.5.
Let's stop users bitching about nightlies pvr addons not working and get some (hopefully good) feedback.
ping @opdenkamp

### CODE REVIEW

**Comment by opdenkamp** [comment]
thanks, but can you squash the bumps and only keep that as a commit and the newly added one as a separate commit

**Comment by hudokkow** [comment]
Squashed

**Comment by opdenkamp** [comment]
jenkins build addons please

**Comment by hudokkow** [comment]
Win32 groovy failure, again.

**Comment by Memphiz** [comment]
thats not a groofy failure ... the printout is missleading ... the error is that nmake stuff and unistd.h missing ...

**Comment by manuelm** [comment]
I'll see if I can fix pvr.filmon for windows

File: project/cmake/addons/addons/pvr.filmon/pvr.filmon.txt [hunk 1]
```diff
@@ -1 +1 @@
-pvr.filmon https://github.com/kodi-pvr/pvr.filmon a0ee270
+pvr.filmon https://github.com/kodi-pvr/pvr.filmon 14419e3
```
Comment (AlwinEsch) [inline project/cmake/addons/addons/pvr.filmon/pvr.filmon.txt]: Can you swap this to 0e5bad6ae0.

The fix for windows is merged, see https://github.com/kodi-pvr/pvr.filmon/pull/5.

File: project/cmake/addons/addons/pvr.mediaportal.tvserver/pvr.mediaportal.tvserver.txt [hunk 2]
```diff
@@ -1 +1 @@
-pvr.mediaportal.tvserver https://github.com/kodi-pvr/pvr.mediaportal.tvserver 87422e6
\ No newline at end of file
+pvr.mediaportal.tvserver https://github.com/kodi-pvr/pvr.mediaportal.tvserver 606a070
```
Comment (AlwinEsch) [inline project/cmake/addons/addons/pvr.mediaportal.tvserver/pvr.mediaportal.tvserver.txt]: Also here to 51e68c4.

Related to https://github.com/kodi-pvr/pvr.mediaportal.tvserver/pull/2.

**Comment by hudokkow** [comment]
Done

**Comment by opdenkamp** [comment]
jenkins build addons please

**Comment by wsnipex** [comment]
time to get this in
jenkins build addons please

File: tools/depends/target/xbmc-pvr-addons/Makefile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 BUILDDIR := $(shell pwd)
-ADDONS = "pvr.argustv pvr.demo pvr.dvblink pvr.dvbviewer pvr.filmon pvr.hts pvr.iptvsimple pvr.mediaportal.tvserver pvr.mythtv pvr.nextpvr pvr.njoy pvr.vdr.vnsi pvr.vuplus pvr.wmc"
+ADDONS = "pvr.argustv pvr.demo pvr.dvblink pvr.dvbviewer pvr.filmon pvr.hts pvr.iptvsimple pvr.mediaportal.tvserver pvr.mythtv pvr.nextpvr pvr.njoy pvr.pctv pvr.vdr.vnsi pvr.vuplus pvr.wmc"
```
Comment (wsnipex) [inline tools/depends/target/xbmc-pvr-addons/Makefile]: drop this change please, not needed anymore

**Comment by hudokkow** [comment]
Done!

**Comment by wsnipex** [comment]
jenkins build addons please

**Comment by Montellese** [comment]
@Memphiz: Is the list of successfully built and failed addons only displayed if at least one addon failed? Might be useful to always show the list of successfully built addons.

**Comment by Memphiz** [comment]
@Montellese not sure what is going on - the workspaces don't have .failure or .success files - they are shown only when one of them or both are there. They didn't seem to be created at all now ...

**Pull request merged**